core/field_label.js

```diff
@@ -43,13 +43,26 @@ goog.require('Blockly.utils.Size');
  * @constructor
  */
 Blockly.FieldLabel = function(opt_value, opt_class) {
-  this.size_ = new Blockly.utils.Size(0, Blockly.Field.TEXT_DEFAULT_HEIGHT);
-  this.class_ = opt_class;
-  opt_value = this.doClassValidation_(opt_value);
-  if (opt_value === null) {
+  if (opt_value == null) {
     opt_value = '';
   }
-  this.setValue(opt_value);
+  Blockly.FieldLabel.superClass_.constructor.call(
+      this, opt_value, null);
```

Comment:
Nit: `opt_value || ''` instead? - Unless we want to accept false and convert it to a string.

Reply:
Hmm, I think this is intentional, so you can do things like: 
``new Blockly.FieldLabel(false)`` and have the value be ``false`` (which is then converted to the string "false" when rendered).
Otherwise the above would yield an empty string.